helix-term/src/commands.rs

```diff
@@ -2251,6 +2281,13 @@ mod cmd {
             fun: open,
             completer: Some(completers::filename),
         },
+        TypableCommand {
+          name: "buffer-close",
+          aliases: &["bc", "bclose"],
+          doc: "Close the current buffer.",
+          fun: buffer_close,
+          completer: None, // FIXME: buffer completer
```

Comment:
I noticed that there is no buffer completer implemented yet -- is this because of the space menu's buffer picker? Would a buffer completer be accepted if I hypothetically contributed it some time in the future?

Reply:
That would be great! I think we had no commands that took a buffer before. I also wasn't sure, should we allow specifying buffers by path, or by an ID (or both)? After https://github.com/helix-editor/helix/commit/e2560f427ef5e75155071e39da342628f5d5896a it's now possible to get the actual numeric ID, and we could display it in the buffer picker too.

Reply:
I opened https://github.com/helix-editor/helix/issues/1043 so as not to bog this specific PR down.